.gitignore

```diff
@@ -22,3 +22,4 @@ genesis-wasm
 
 # nix output result symbol link
 result
+.direnv/
```

Comment:
Please put this modification in another `git commit` with commit message: "chore: update `.gitignore`"